firmware/application/apps/ui_playlist.cpp

```diff
@@ -91,8 +94,10 @@ void PlaylistView::on_file_changed(std::filesystem::path new_file_path, rf::Freq
     File data_file;
     // Get file size
     auto data_open_error = data_file.open("/" + new_file_path.string());
-    if (data_open_error.is_valid()) {
-        file_error();
+    if (!data_open_error.is_valid()) {
+        track_number++;
+    } else if (data_open_error.is_valid()) {
+        file_error("C16 file\n" + new_file_path.string() + "\nread error.");
```

Comment:
The path is included in the file_error already? Do you want it here too?

Reply:
They are actually different cuz the one ( ``new_file_path`` ) in the file_error() is the one that been used before ``on_filechanged`` were called so it's been overloaded once.  
For example if i manually create a error, here it shows. 
![IMG_0245](https://github.com/eried/portapack-mayhem/assets/24917424/e094811f-2c41-4100-8d38-10f4f226f46e)